production/helm/loki/values.yaml

```diff
@@ -212,6 +212,7 @@ loki:
       s3ForcePathStyle: false
       insecure: false
       http_config: {}
+      use_irsa: true
```

Comment:
this would be a breaking change no?. Breaks the existing helm users who uses ACCESSKEY based S3 credentials.

I would prefer to keep `false` as default and document this additional configs on the S3Config page (use_irsa + additional ENVs that need to be set)
https://grafana.com/docs/loki/latest/configuration/#s3_storage_config